notebooks/example_create_agent.ipynb

```diff
@@ -0,0 +1,284 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Example Create Agent using Gen AI SDK"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Authors**: \n",
+    "- Muhammad Afif Al hawari (muhammad.a.a.hawari@gdplabs.id)\n",
+    "\n",
+    "**Reviewers**: \n",
+    "- Villia Susanti (villia.susanti@gdplabs.id)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Summary\n",
+    "In this Notebook we will try to create a sample agent using Gen AI SDK and Open AI GPT 3.5. The core idea of agents is to use a language model to choose a sequence of actions to take. The Agent will use two tools, one is Search Tool which utilize google search API and multiplier tool which takes two numbers and multipliy them.\n",
+    "\n",
+    "## References\n",
+    "- [Langchain Agent Docs](https://python.langchain.com/docs/modules/agents/)\n",
+    "- [Gen AI Agent Orchestrator](https://docs.glair.ai/generative-internal/modules/agent-orchestrator)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prepare Environment"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Install Dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pip install google-search-results"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Install Gen AI SDK\n",
+    "\n",
+    "To install the SDK library, we need to create a personal access token on GitHub. Please follow these steps:\n",
+    "1. Log in to your [GitHub Account](https://github.com/).\n",
+    "2. Go to the [Personal Access Tokens](https://github.com/settings/tokens) page.\n",
+    "3. If you haven't created a Personal Access Tokens yet, you can generate one.\n",
+    "4. When generating a new token, make sure that you have checked the `repo` option to grant access to private repositories.\n",
+    "5. Now, you can copy the new token that you have generated and paste it into the script below."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import getpass\n",
+    "import subprocess\n",
+    "import sys\n",
+    "import os\n",
+    "\n",
+    "def install_sdk_library():\n",
+    "    token = getpass.getpass(\"Input Your Personal Access Token: \")\n",
+    "\n",
+    "    cmd = f\"pip install -e git+https://{token}@github.com:GDP-ADMIN/gen-ai-internal.git#egg=gdplabs_gen_ai\"\n",
```

Comment:
I can only execute this function after change the cmd like below
cmd = f"pip install -e git+https://{token}@github.com/GDP-ADMIN/gen-ai-internal.git#egg=gdplabs_gen_ai"

if not I got this error:
 fatal: unable to access 'https://github.com:GDP-ADMIN/gen-ai-internal.git/': URL rejected: Port number was not a decimal number between 0 and 65535
 
 After changed it, I can install the library
```
....
Successfully built gdplabs_gen_ai
Installing collected packages: gdplabs_gen_ai
  Attempting uninstall: gdplabs_gen_ai
    Found existing installation: gdplabs-gen-ai 0.0.1b6
    Uninstalling gdplabs-gen-ai-0.0.1b6:
      Successfully uninstalled gdplabs-gen-ai-0.0.1b6
Successfully installed gdplabs_gen_ai-0.0.1b6
```

Reply:
This should be because the SDK code still not in branch main yet.

You can change it to for the time being
```python
cmd = f"pip install -e git+https://{token}@github.com/GDP-ADMIN/gen-ai-internal.git@agent-tool#egg=gdplabs_gen_ai"
```

Reply:
It is recommended to use 
```sh
pip install "gdplabs_gen_ai @ git+https://<token>@github.com/GDP-ADMIN/gen-ai-internal.git"
```
instead to avoid deprecation warning when using 

> https://pip.pypa.io/en/stable/news/#id111
> Deprecated a historical ambiguity in how egg fragments in URL-style requirements are formatted and handled. egg fragments that do not look like [PEP 508](https://peps.python.org/pep-0508/) names now produce a deprecation warning. ([#11617](https://github.com/pypa/pip/issues/11617))